.github/workflows/build_and_test.yml

```diff
@@ -174,8 +174,9 @@ jobs:
       run: |
         # Hive "other tests" test needs larger metaspace size based on experiment.
         if [[ "$MODULES_TO_TEST" == "hive" ]] && [[ "$EXCLUDED_TAGS" == "org.apache.spark.tags.SlowHiveTest" ]]; then export METASPACE_SIZE=2g; fi
```

Comment:
Other tests don't need higher METASPACE_SIZE, so I keep smaller METASPACE_SIZE for others, only high METASPACE_SIZE for Hive test.